pkg/common/monitoring/monitoring.go

```diff
@@ -44,7 +44,7 @@ const (
 	RemoteWriteTimeout             = "60s"
 
 	OrgIDHeader        = "X-Scope-OrgID"
-	DefaultWriteTenant = "anonymous"
+	DefaultWriteTenant = "giantswarm"
 )
 
 var DefaultReadTenants = []string{"anonymous"}
```

Comment:
```suggestion
var DefaultReadTenants = []string{"anonymous", "giantswarm"}
```

Reply:
Didn't we say we would have `giantswarm` as the default tenant, but create an extra datasource for `anonymous` metrics?

Reply:
This is currently used by the sharding code to query Mimir

Reply:
The datasources have some hardcoded stuff here now https://github.com/giantswarm/observability-operator/pull/282/files#diff-fbd05bc15d97df57880fa9778e36e8c931d40b7824eece692af32065949fc921L46

The mimir and mimir-alertmanager DS will use the tenant configured in the `shared-org` and `giantswarm` grafana organization CR which is giantswarm